src/include/ipxe/netdevice.h

```diff
@@ -426,6 +426,12 @@ struct net_device {
 
 	/** Network device configurations (variable length) */
 	struct net_device_configuration configs[0];
+
+	/** Static routes from DHCP option 121 */
+	uint8_t * static_routes;
+
+	/** DHCP option 121 length */
+	uint8_t static_routes_len;
```

Comment:
You cannot put IPv4-specific fields in the protocol-independent `struct net_device`.

Also, the variable-length `configs[0]` field must be at the end of the structure, as with all variable-length fields in C.